paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -210,12 +204,12 @@ internal class CustomerSheetViewModel @Inject constructor(
         }
     }
 
-    private fun onFormValuesChanged(formFieldValues: FormFieldValues?) {
-        val isComplete = formFieldValues?.fieldValuePairs?.all {
-            it.value.isComplete
-        }
+    private fun onFormDataUpdated(formData: FormViewModel.ViewData) {
         updateViewState<CustomerSheetViewState.AddPaymentMethod> {
-            it.copy(primaryButtonEnabled = isComplete ?: false)
+            it.copy(
+                formViewData = formData,
+                primaryButtonEnabled = formData.completeFormValues != null
```

Comment:
Should this be a computed property on the view state now?

Reply:
I don't think so, the view state only has the current form data that is updated with the FormViewModel form data flow.